src/plugins/intel_gpu/include/intel_gpu/primitives/resample.hpp

```diff
@@ -123,6 +123,36 @@ struct resample : public primitive_base<resample> {
             throw std::runtime_error("Resample's scales/axes count does not match");
     }
 
+    /// @brief resample with non-constant scales
+    resample(const primitive_id& id,
+             const primitive_id& input,
+             const std::vector<int64_t>& sizes,
```

Comment:
isnt it better to have one unified primitive API with vector<primitive_id> inputs and then process each cases w.r.t. the # of inputs?